great_expectations/core/util.py

```diff
@@ -243,6 +266,14 @@ def ensure_json_serializable(data):
             for idx, val in data.iteritems()
         ]
         return
+
+    elif pyspark and isinstance(data, pyspark.sql.DataFrame):
+        # using StackOverflow suggestion for converting pyspark df into dictionary
+        # https://stackoverflow.com/questions/43679880/pyspark-dataframe-to-dictionary-columns-as-keys-and-list-of-column-values-ad-di
+        return ensure_json_serializable(
+            dict(zip(data.schema.names, zip(*data.collect())))
+        )
+
```

Comment:
This looks same as in lines 152 -- 157...

Reply:
the function calls are slightly different here `ensure_json_serializable()` vs `convert_to_json_serializable()`, but yes the inner function-call is the same. Would you prefer to have this separated out?